app/src/main/res/values/button_themes.xml

```diff
@@ -4,6 +4,7 @@
     <style name="Hedvig.Button" parent="Widget.MaterialComponents.Button">
         <item name="backgroundTint">?colorButton</item>
         <item name="android:textColor">?textColorButton</item>
+        <item name="colorButton">@color/colorLargeButton</item>
```

Comment:
Should this color simply be called `colorButton`?

Reply:
We need to talk to Zak about that because now only the large buttons are purple

Reply:
I've checked with Zak. All buttons on dark should be the purple-ish color